docs/changelog-assets/README.md

```diff
@@ -0,0 +1,46 @@
+# Changelog Assets
+
+This directory contains all assets used in the [CHANGELOG](../../CHANGELOG.md).
+
+## Adding Assets
+
+### Naming
+
+Files should be hashed and named after their hash (`{hash}.{ext}`). This has multiple benefits:
+
+- Prevents duplicate files.
+- Prevents duplicate file names.
+- Prevents uninformative file names.
+
+The files in this directory aren't going to be looked up manually. They will be referenced by the CHANGELOG. As such, craming information into the file name is useless and will just lead to weird names and a complete mess.
+
+For hashing, we'll use Blake2b with a length of 128. This hash can be computed using the [`cksum`](https://www.gnu.org/software/coreutils/manual/html_node/cksum-invocation.html#cksum-invocation) coreutils program:
+
+```bash
+cksum -a blake2b --untagged --length=128 filename
+```
+
+### Images
+
+- Images should be cropped to the contents, don't upload full screenshots.
+- Images should be uploaded in the WebP format with the following options:
+    - Image quality: 80%
+    - Alpha quality: 100%
+    - No Exif data
+    - No IPTC
+    - No XMP data
+    - With color profile
+    - No thumbnail
+
+You can use any image manipulation program for this, like GIMP. Otherwise, if the image has already been cropped and you want to convert it to a WebP, use ffmpeg:
+
+```bash
+ffmpeg -i input.png -pix_fmt yuv420p -c:v libwebp -quality 80 -compression_level 6 output.webp
+```
+
+### Videos
+
+- Videos should be cropped to contents, don't record your entire desktop.
+
+TODO: decide on a format, probably WebM for longer videos and animated WebP for shorter videos. Definitely won't be using GIFs.
+
```

Comment:
This is a PITA. Adding an image to the changelog could be as simple as ctrl-v on a wiki page or on the release description page. 
I'm fine with the guidelines on the contents of the image, but asking to run ffmpeg on them and check summing them makes this simply ugh.

Reply:
I was planing on adding a script as a one-click solution but didn't want to put that in this PR.

Reply:
I think I'm less fine now about committing the changelog to the repo. This means also committing all the images and videos. Those could instead be put in the wiki on on the release page description without issues and without growing the repo itself.

Reply:
Those four images are 67 KB in size. That's 16.75 KB per image. Even if you have one thousand of those images, they'd only take up 16 MB.